src/components/ChatLog.test.js

```diff
@@ -5,26 +5,31 @@ import { render, screen } from "@testing-library/react";
 
 const LOG = [
   {
+    id: 1,
```

Comment:
The intention was that you would derive the ids from the index of the object, not hard code them.